examples/pennylane_run_doubly_stochastic.py

```diff
@@ -0,0 +1,312 @@
+r"""
+Doubly stochastic gradient descent
+==================================
+
+In this tutorial we investigate and implement the doubly stochastic gradient descent
+paper from `Ryan Sweke et al. (2019) <https://arxiv.org/abs/1910.01155>`__. In this paper,
+it is shown that quantum gradient descent, where a finite number of measurement samples
+(or *shots*) are used to estimate the gradient, is a form of stochastic gradient descent.
+Furthermore, if the optimization involves a linear combination of expectation values
+(such as VQE), sampling from the terms in this linear combination can further reduce required
+resources, allowing for "doubly stochastic gradient descent".

```

Comment:
Can we mention related work? It's only fair, and it's a friend of mine :) 

 

```suggestion
resources, allowing for "doubly stochastic gradient descent".

Note that based on very similar observations, `Jonas Kuebler et al. (2019)` 
recently proposed an optimizer (which they call the *individual Coupled Adaptive 
Number of Shots (iCANS)* optimizer) that adapts the shot number of 
measurements during training.
```